person.rb

```diff
@@ -25,4 +26,8 @@ def correct_name
     super
     @name
   end
+
+  def add_rental(book, date)
+    Rentals.new(book, date, self)
```

Comment:
- [ ] This should be `Rental` (singular), not `Rentals` (plural)

- [ ] Kindly maintain the order of the arguments.

  ```
  Rental.new(date, book, self)
  ```